app/main.py

```diff
@@ -1 +1,24 @@
-# write your code here
+
+from app.cafe import Cafe
+
+from app.errors import (VaccineError, NotWearingMaskError)
```

Comment:
Place each Exception on a new line as it is done in `app/cafe.py` in case you wrap them in parenthesis.